### ISSUE TITLE
[FIX] titles & meta descriptions

### ISSUE THREAD

**Issue by matthewlilley** [sushiswap/sushiswap-interface#295]
Each page has a title & meta description included in the page as can be seen here (https://github.com/sushiswap/sushiswap-interface/blob/canary/src/pages/tools/meowshi.tsx#L136), however, when linking the default is always shown. 

This needs some investigation and a resolution. At minimum we should be able to set title, meta description & image on a per page basis.

**Comment by ranjithjupaka** [comment]
i would like to fixthis

**Comment by matthewlilley** [comment]
Absolutely @ranjithjupaka, please let me know what you discover!

**Comment by chxash** [comment]
Problem seems to be fixed ?
![title](https://user-images.githubusercontent.com/5720927/134006405-03546a89-d05d-4abb-a021-b5513babc8bb.png)
![title2](https://user-images.githubusercontent.com/5720927/134006416-b30444c1-41d1-45e5-a858-2eec84aaa423.png)

![meta](https://user-images.githubusercontent.com/5720927/134006426-93f12a70-8a00-48de-9cce-8565786f8fe7.png)

**Comment by matthewlilley** [comment]
> Problem seems to be fixed ? ![title](https://user-images.githubusercontent.com/5720927/134006405-03546a89-d05d-4abb-a021-b5513babc8bb.png) ![title2](https://user-images.githubusercontent.com/5720927/134006416-b30444c1-41d1-45e5-a858-2eec84aaa423.png)
> 
> ![meta](https://user-images.githubusercontent.com/5720927/134006426-93f12a70-8a00-48de-9cce-8565786f8fe7.png)

When shared via social like discord there is still an issue. Would be great to fix this.

**Comment by chxash** [comment]
After investigating a little bit, I found that twitter bot / parser for twitter cards is not able to fetch our updated titles / descriptions.
A good way to have an idea of what the twitter bot is fetching is to directly use that kind of command : "curl -A twitterBot url".

I have tried various ways to fix this, I was not able to make it work by just using next/head capabilities.
One way that is working for me is to pass meta information through "getStaticProps" method in related components.
I will submit code for this when I have time and maybe it will spark discussion.
